site/content/blogs/release-v121.en.md

```diff
@@ -28,12 +28,53 @@ Copilot v1.21 brings several new features and improvements:
 ## CloudFront Integration
 
 ## Configure Environment Security Group
-
+You can now configure your environment security group rules through environment manifest.   
+Sample security group rules template inside environment manifest is given below.
+```yaml
+network:
+  vpc:
+    security_group:
+      ingress:
+        - ip_protocol: tcp
+          ports: 0-65535
+          cidr: 0.0.0.0/0
+      egress:
+        - ip_protocol: tcp
+          ports: 80
+          cidr: 0.0.0.0/0
+```
 ## `job logs`
 
 ## Package Addons CloudFormation Templates
 
-## ELB Access Log Support
+## ELB Access Logs Support
+You can now enable Elastic Load Balancing access logs that capture detailed information about requests sent to your load balancer.
+There are a few ways to enable access logs: 
+
+1. You can specify `access_logs: true` in your environment manifest as shown below and Copilot will create an S3 bucket where the Public Load Balancer will store access logs.
+```yaml
+name: qa
+type: Environment
+
+http:
+  public:
+    access_logs: true 
+```
+
+2. You can also bring in your own bucket and prefix. Copilot will use those bucket details to enable access logs. 
+You can do that by specifying the following configuration in your environment manifest.
+```yaml
+name: qa
+type: Environment
+
+http:
+ public:
+   access_logs:
+     bucket_name: my-bucket
+     bucket_prefix: my-prefix
```

Comment:
Is it just `prefix` now? 
```suggestion
     prefix: my-prefix
```

Reply:
Yes, so before making changes here I made sure to add `prefix` in the implementation part so sent out a other PR. Now I will make it `prefix`.